src/CTA.WebForms/Extensions/RegexExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace CTA.WebForms.Extensions
+{
+    public static class RegexExtensions
+    {
+        /// <summary>
+        /// Allows for regex replacements using async function delegates.
+        /// </summary>
+        /// <param name="regex">The regular expression that is being used.</param>
+        /// <param name="input">The string that <paramref name="regex"/> is being applied to.</param>
+        /// <param name="replacementFn">The function that is used for match replacement.</param>
+        /// <returns>The string <paramref name="input"/> with all matches made by <paramref name="regex"/>
```

Comment:
just specify return value, it is confusing when you include paramref name="input" in the comments